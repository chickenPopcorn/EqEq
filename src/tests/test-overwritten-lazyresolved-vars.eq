MyCtx = {
  a = { b + 5; }
}

MyCtx: find a {
  b = 5;
  print("a = %.0f (when b = %.0f)\n", a, b);

  b = 20;
  print("a = %.0f (when b = %.0f)\n", a, b);

  c = 100;
  a = c;
  print("a = c = %.0f\n", a);
}
