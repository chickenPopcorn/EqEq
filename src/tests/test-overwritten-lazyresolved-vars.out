a = 10 (when b = 5)
a = 25 (when b = 20)
a = c = 100
